iFixit/react-commerce
# Vulcan: Make Causes list visible above the fold

This does what it says on the tin.

## QA Notes
Please confirm that it handles a page with no introduction text correctly (it should hide the link to the introduction), and that the icon is clear to you. Detailed spec: https://docs.google.com/document/d/1YhlRLq1C6sRU_nYC-Kf9eQ6CDGvjHZU_5zwo3NIqLuI/edit#bookmark=id.vyolc43ijwj

Closes https://github.com/iFixit/ifixit/issues/49392
